Marlin/src/lcd/menu/menu_tramming.cpp

```diff
@@ -99,4 +96,4 @@ void goto_tramming_wizard() {
   });
 }
 
-#endif // HAS_LCD_MENU && ASSISTED_TRAMMING_WIZARD
+#endif // BOTH(HAS_LCD_MENU, ASSISTED_TRAMMING_WIZARD)
```

Comment:
I don't think a comment needs that change.

Reply:
It really doesn't matter either way. I don't like closing comments at all, but if we are going to have them I think they should match.

Reply:
Are you planning to change all the comments in marlin to that? Because I only see one like that:
https://github.com/MarlinFirmware/Marlin/blob/4402a0578a9fa5c9743eb5774224ce206c618ce9/Marlin/src/lcd/extui/lib/ftdi_eve_touch_ui/screens/leveling_menu.cpp#L121